02_munge/src/salt_front_record.R

```diff
@@ -0,0 +1,25 @@
+#===================================================================================#
```

Comment:
Should this file get added into the Snakefile for fetch/munge?

Reply:
Yes, it should, I'll open an issue for that.